doc/content/guides/getting-started/configuration.md

```diff
@@ -56,6 +56,7 @@ stack:
   volumes:
     - './acme:/var/lib/acme'
     - './data/blob:/srv/ttn-lorawan/public/blob'
+    - './ttn-lw-stack.yml:/.ttn-lw-stack.yml:ro'
```

Comment:
Doing this might lead to some confusion, since Docker won't pick up on updates when making changes to the file. Maybe better to put it in a dir

Reply:
Updated to use a dir-mount. Moved stack config to `config/stack/ttn-lw-stack.yml`